packages/integration-tests/contracts/ERC20.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+contract ERC20 {
```

Comment:
We could technically delete this and just import the oz erc20 JS npm package if we wanted to

Reply:
Can we compile the NPM package with Forge without having to resort to more submodules?

Reply:
`forge` has a concept of remappings in `foundry.toml` where you say the `name-of-import=path-to-file`

There is also the `--hardhat` flag

Reply:
Not the biggest deal, but an easy way to remove code